### PR TITLE
Added caching 404s for 1 minute

### PR DESCRIPTION
**Pull request by jondade**

### CODE REVIEW

File: etc/fastly.vcl [hunk 1]
```diff
@@ -94,6 +94,19 @@ sub vcl_recv {
 }
 
 sub vcl_fetch {
+
+    # Remove Set-Cookies from responses for static content
+    # to match the cookie removal in recv.
+    if (req.url ~ "^/(pub/)?(media|static)/.*") {
```
Comment (acme) [inline etc/fastly.vcl]: You don't need the `.*` there.
Is this the correct path for assets?

File: etc/fastly.vcl [hunk 2]
```diff
@@ -94,6 +94,19 @@ sub vcl_recv {
...
+        unset beresp.http.set-cookie;
+
+        # Set a short TTL for 404's
+        if (beresp.status == 404) {
+            set beresp.ttl = 300s;
```
Comment (acme) [inline etc/fastly.vcl]: 5 minutes seems quite long when waiting for a missing asset to work again.

Comment (jondade) [inline etc/fastly.vcl]: Its static assets in magento they are unlikely to be added sooner and the 404 can be purged. This felt like the balance of offloading refreshes of missing assets.

File: etc/fastly.vcl (hunk 1 above)
Comment (jondade) [inline etc/fastly.vcl]: Yup. Fair point.

**Comment by acme** [comment]
Reviewed and approved.

**Pull request merged**